sdk/keyvault/tests.yml

```diff
@@ -1,7 +1,7 @@
 trigger: none
 
 stages:
-  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
Key Vault uses templates for each of its packages (keys, secrets, and certificates) -- would this change, and the removal of `EnvVars`, need to be made for each of them?

Reply:
Ack, looks like this file is incomplete, must have missed a couple changes there. Good catch. I'll normalize all the paths and add the env vars back to the certificates tests.